sources/packages/backend/libs/sims-db/src/entities/application-disability-status.type.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Application disability status.
+ */
+export enum ApplicationDisabilityStatus {
+  yes = "yes",
+  noIWantToAccessOtherFundingTypes = "noIWantToAccessOtherFundingTypes",
+  noIDoNotHaveADisability = "noIDoNotHaveADisability",
+}
+
+/**
+ * Application disability status description.
+ */
+export const ApplicationDisabilityStatusDescription: Record<
```

Comment:
IMO the conversion is not required from API perspective. We can do this at Vue level.

Reply:
IMO it should be in only one place and with the API sending that to the view we can avoid duplicating this logic in the view.